Python/Temperature_Conversion/Temperature_Conversion.py

```diff
@@ -1,6 +1,6 @@
 """ TEMPERATURE CONVERSION """
 
-#to convert from one scale to another
+'''to convert from one scale to another'''
```

Comment:
Thats not how DocStrings work

Check the styling guidelines [here](https://github.com/HarshCasper/Rotten-Scripts/blob/master/STYLE_GUIDELINES.md#python)